tutorials/k8s-kapsule-multi-az/index.mdx

```diff
@@ -0,0 +1,625 @@
+    ```
+    data "template_file" "kubeconfig" {
+        template = <<EOT
+    apiVersion: v1
+    clusters:
+    - name: "$${cluster_name}"
+    cluster:
+        certificate-authority-data: $${ca_cert}
+        server: "$${host}"
+    contexts:
+    - name: admin@kapsule-multi-az
+    context:
+        cluster: "kapsule-multi-az"
+        user: kapsule-multi-az-admin
+    current-context: admin@kapsule-multi-az
+    kind: Config
+    preferences: {}
+    users:
+    - name: kapsule-multi-az-admin
+    user:
+        token: $${token}
+    EOT
+
+        vars = {
+            cluster_name = scaleway_k8s_cluster.kapsule_multi_az.name
+            host         = scaleway_k8s_cluster.kapsule_multi_az.kubeconfig[0].host
+            token        = scaleway_k8s_cluster.kapsule_multi_az.kubeconfig[0].token
+            ca_cert      = scaleway_k8s_cluster.kapsule_multi_az.kubeconfig[0].cluster_ca_certificate
+        }
+    }
+
+    resource "local_file" "kubeconfig" {
+        content  = data.template_file.kubeconfig.rendered
+        filename = "${path.module}/kubeconfig"
+    }
+    ```
+
+6. Initialize your Terraform configuration and apply it to create your resources.
+    ```
+    terraform init
+    terraform apply
+    ```
+
+7. Configure your `KUBECONFIG` environment variable to point to the `kubeconfig` file you just created via Terraform
+    ```
+    export KUBECONFIG=kubeconfig
+    ```
+
+8. List your nodes now to check that your cluster is ready, and that there are two nodes per availability zone.
+    ```
+    kubectl get nodes
+
+    NAME                                             STATUS   ROLES    AGE   VERSION
+    scw-kapsule-multi-az-pool-fr-par-1-61e22198f8c   Ready    <none>   89s   v1.28.0
+    scw-kapsule-multi-az-pool-fr-par-1-8334e772ced   Ready    <none>   82s   v1.28.0
+    scw-kapsule-multi-az-pool-fr-par-2-1bcf90f3683   Ready    <none>   90s   v1.28.0
+    scw-kapsule-multi-az-pool-fr-par-2-33265e85597   Ready    <none>   86s   v1.28.0
+    scw-kapsule-multi-az-pool-fr-par-3-44b14b7bbbd   Ready    <none>   84s   v1.28.0
+    scw-kapsule-multi-az-pool-fr-par-3-863491657c7   Ready    <none>   80s   v1.28.0
+    ```
+
+## Nginx Ingress Controller as a stateless multi-az application
+
+A very common pattern is to deploy an Ingress Controller to externally expose applications hosted inside the Kapsule cluster. Configuring a multi-AZ deployment of your Ingress Controller will greatly improve its resilience and scalability.
+
+In this part we will deploy an [ingress-nginx](https://github.com/kubernetes/ingress-nginx) in a multi-AZ, production ready, configuration, using the previously created Kapsule cluster. The official Helm for the nginx controller is full-featured, so we will use it through the Terraform Helm provider.
+
+* Healthy cluster
+<Lightbox src="scaleway-k8s-kapsule-multi-az-2.webp" alt="" size="large" />
```

Comment:
Pixeled on my preview site, small parts are almost unreadable.
- Anything we can do to improve the quality? 
I exported the webp from draw.io directly, perhaps there's better compression elsewhere. 

- Do we really need the lightbox? It's not enlarging much.